src/Microsoft.SqlTools.ServiceLayer/Workspace/WorkspaceService.cs

```diff
@@ -94,6 +96,13 @@ public WorkspaceService()
         public delegate Task TextDocCloseCallback(ScriptFile closedFile, EventContext eventContext);
 
         /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="openFile"></param>
+        /// <param name="eventContext"></param>
+        public delegate Task RebuildIntelliSenseCallback(ScriptFile openFile, EventContext eventContext);        
```

Comment:
This isn't needed - per my other comment, we should handle the event directly in the LanguageService